.github/workflows/engine_pr_test.yml

```diff
@@ -101,6 +101,11 @@ jobs:
           source ~/openquake/bin/activate
           pip install pytest https://wheelhouse.openquake.org/v3/py/rtgmpy-1.0.0-py3-none-any.whl
           oq engine --upgrade-db
+          oq webui start 127.0.0.1:8800 -s &
```

Comment:
We might add a comment like:
`Starting the webui here is not needed by test_public_mode, but here we also want to check that the webui starts without errors`